lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -279,9 +289,44 @@ int main(int argc, char* argv[]) {
   magma::SessionProxyResponderAsyncService proxy_service(
       server.GetNewCompletionQueue(), std::move(proxy_handler));
   server.AddServiceToServer(&local_service);
+  MLOG(MINFO) << "Add localservice";
   server.AddServiceToServer(&proxy_service);
+  MLOG(MINFO) << "Add proxyservice";
+  std::thread access_common_message_thread;
+  /* Initialize the main thread of session managment by folly event to handle
+   * logical component of 5G of SessionD
+   */
+  auto session_enforcer = std::make_shared<magma::SessionStateEnforcer>(
+                            rule_store, *session_store, pipelined_client,
+	                        amf_srv_client, mconfig);
+  //5G related async msg handler service framework creation
+  auto set_message_handler = std::make_unique<magma::SetMessageManagerHandler>(
+		  session_enforcer, *session_store);
+  MLOG(MINFO) << "session enforcer";
+  //5G specific services to handle set messages from AMF and mme
+  magma::AmfPduSessionSmContextAsyncService set_message_service(
+		  server.GetNewCompletionQueue(), std::move(set_message_handler));
+  MLOG(MINFO) << "Amfpdusessionsmcontext set message started";
+    //5G related services
+  if (converged_access)
+  {
+    server.AddServiceToServer(&set_message_service);
+    MLOG(MINFO) << "Add converged access GRPC message service";
```

Comment:
Can you either move this log before the server is started or change it to 
"Added"

Reply:
Changed